modules/ocf_desktop/files/xsession/notify

```diff
@@ -32,15 +31,10 @@ def main():
         ).show()
 
     # Show user's print quota when changed
-    old_quota = None
-    while True:
-        quota = subprocess.check_output(PAPER_PATH)
-        if quota != old_quota:
-            title, msg = quota.decode(encoding='UTF-8').split('\n', 1)
-            msg += '(as of {})'.format(datetime.now().strftime('%-I:%M%P'))
-            Notify.Notification.new(title, msg, NOTIFICATION_ICON).show()
-            old_quota = quota
-        time.sleep(60)
+    quota = subprocess.check_output(PAPER_PATH)
+    title, msg = quota.decode(encoding='UTF-8').split('\n', 1)
+    msg += '(as of {})'.format(datetime.now().strftime('%-I:%M%P'))
+    Notify.Notification.new(title, msg, NOTIFICATION_ICON).show()
```

Comment:
i still think this should use the ocflib functions rather than parsing the paper output

Reply:
Understandable, but this is how notify was; I just took out the infinite loop. It makes more sense to do this in a separate PR.

Reply:
agreed